ase/Makefile

```diff
@@ -447,9 +466,7 @@ help: header
 	@echo "#################################################################"
 
 ## Build ASE Software objects and shared library ##
-sw_build:
-	make header
-	mkdir -p $(WORK)
+sw_build: $(SIM_QUARTUS_VERILOG) $(SIM_QUARTUS_VHDL)
```

Comment:
Why sw_build depends on $(SIM_QUARTUS_VERILOG) and $(SIM_QUARTUS_VHDL) here?

Reply:
To avoid a race on creating the $(WORK) directory.